dev/io.openliberty.checkpoint_fat_persistence/bnd.bnd

```diff
@@ -0,0 +1,80 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/db2fat/src
+
+fat.project: true
+
+fat.test.container.images: kyleaure/db2-ssl:3.0
+
+# Define additional tested features that are NOT present in any XML files in this bucket.
+# In this case, osgiConsole-1.0 is added programmatically at runtime.
+tested.features:\
```

Comment:
Probably don't need this if you are not programmatically adding features to the server.xml